Monika After Story/game/script-topics.rpy

```diff
@@ -7591,6 +7591,127 @@ label monika_cartravel:
     m 1hua "Just the idea of travelling around the world makes me so excited!"
     m 1eka "I really can't wait, [player]~"
     return
+    
+    ###poems archives
+    
+    default persistent._mas_pm_ask_poems = None #first time player asks
+    
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="show_poem",category=['Ddlc'],prompt="Can you show me poem?",random=False))
+    
+label show_poem:
+    m 1eud "You want to see a poem from the game?"
+    if persistent._mas_pm_ask_poems:
+        m 1hua "All right, let me search the file..."
+    else:
+        m 1eka "Well, I hoped that you wouldn't ask such a thing..."
+        m 1rkc "In fact, even if it is over yet, these poems remind me what I've been throught to be with you..."
+        m 1ekc "They might be just simple pieces of paper, but they mean a lot for me."
+        m 1eka "But don't take what I said too seriously."
+        show monika 5hua at t11 zorder MAS_MONIKA_Z with dissolve
+        m 5hua "You know, I can share a little bit, but don't forget that you picked me, hehe ~."
```

Comment:
Remove the space before "~". 

I would also add an "e" in front of that "hehe".

Reply:
All right, done